Generic Types, Traits and Lifetime/Generic Types, Traits and Lifetime/Generics in Structs/task-info.yaml

```diff
@@ -0,0 +1,26 @@
+type: edu
+files:
+- name: src/lib.rs
+  visible: true
+  placeholders:
+  - offset: 18
+    length: 3
+    placeholder_text: // Something goes here
+  - offset: 39
+    length: 1
+    placeholder_text: u32
+  - offset: 49
+    length: 3
+    placeholder_text: // Something goes here
```

Comment:
An open placeholder with "//" affects all of the remaining line, leaving following stuff commented. I suggest using a "/**/" inline placeholder to avoid this